package.json

```diff
@@ -36,7 +36,8 @@
         "will-call": "1.x.x"
     },
     "peerDependencies": {
-        "typescript": ">=3.6.5"
+        "typescript": ">=3.6.5",
+        "@hapi/eslint-plugin": "^5.1.0"
```

Comment:
I’m not sure how it exactly will work with npm, but I suspect setting it as “*” might work better.

Reply:
I thought of doing it but I wasn't sure either and thought of mimicking the version requirement of the `dependencies` field